src/components/application_manager/src/request_controller.cc

```diff
@@ -475,9 +475,12 @@ void RequestController::Worker::exitThreadMain() {
 void RequestController::UpdateTimer() {
   LOG4CXX_AUTO_TRACE(logger_);
   RequestInfoPtr front = waiting_for_response_.FrontWithNotNullTimeout();
+  // Buffer for sending request
+  const uint32_t delay_time = 100u;
```

Comment:
@OHerasym Because 50 ms can be not enough and 200 ms is too much for transporting message from SDL to HMI.